modules

```diff
@@ -9,7 +9,8 @@ PACKAGES_OPENWRT_BRANCH=lede-17.01
 PACKAGES_OPENWRT_COMMIT=82ef2fd773987a72678a32b862f50085c3f57d56
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
-PACKAGES_GLUON_COMMIT=57c67964e2c0e7604464f8b77e5ffbba2f8ab9ba
+PACKAGES_GLUON_COMMIT=7e4e096bc16ce8c3b1b4561fe530b155519f963f
+PACKAGES_GLUON_BRANCH=l3roamd_bump
 
```

Comment:
of course this will have to point to the master branch. l3roamd was a moving target in the last few weeks. I am expecting this to slow down somewhat. I will adjust this commitID shortly before merge.